pyabc/acceptor/acceptor.py

```diff
@@ -0,0 +1,288 @@
+"""
+Acceptor
+--------
+
+After summary statistics for samples for given parameters have
```

Comment:
After summary statistics for samples for given parameters have --> After summary statistics for samples **of** given parameters have ?